manifests/dhcp6.pp

```diff
@@ -0,0 +1,68 @@
+# class dhcp::dhcp6
+class dhcp::dhcp6 (
+  Array[String] $nameservers = ['2001:4860:4860::8888', '2001:4860:4860::8844'],
+  Integer[0] $default_lease_time6 = 2592000,
+  Integer[0] $default_preferred_lifetime6 = 604800,
+  Integer[0] $default_renewal_time6 = 3600,
+  Integer[0] $default_rebinding_time6 = 7200,
+  Integer[0] $default_info_refresh_time6 = 21600,
+  Boolean $leasequery6 = true,
+  Boolean $rapid_commit6 = false,
+  Integer[0] $default_preference6 = 10,
+  Stdlib::Absolutepath $dhcp_dir = $dhcp::params::dhcp_dir,
+  String $dhcp_conf6 = $dhcp::params::dhcp_conf6,
+  String $packagename = $dhcp::params::packagename,
```

Comment:
This is unused. I think both classes should use ```ensure_package``` to make sure both can used independently and at the same time.

Reply:
You are correct, the packagename param is unused, and should be removed from dhcp::dhcp6.